spec/controllers/well_known/webfinger_controller_spec.rb

```diff
@@ -167,5 +169,39 @@
         expect(response).to have_http_status(400)
       end
     end
+
+    context 'when an account has an avatar' do
+      let(:alice) { Fabricate(:account, username: 'alice', avatar: attachment_fixture('attachment.jpg')) }
+      let(:resource) { alice.to_webfinger_s }
+
+      before do
+        perform_show!
+      end
+
+      it 'returns avatar in response' do
+        json = body_as_json
+
+        avatar_link = json[:links].find { |link| link[:rel] == 'http://webfinger.net/rel/avatar' }
+        expect(avatar_link).to_not be_nil
+        expect(avatar_link[:type]).to eq alice.avatar.content_type
+        expect(avatar_link[:href]).to eq full_asset_url(alice.avatar)
+      end
+    end
+
+    context 'when an account does not have an avatar' do
+      let(:alice) { Fabricate(:account, username: 'alice', avatar: nil) }
+      let(:resource) { alice.to_webfinger_s }
+
+      before do
+        perform_show!
+      end
+
+      it 'returns avatar in response' do
```

Comment:
I think the spec below is correct, but the wording on this example may have been copy/pasted in and is not accurate?

Reply:
yep you're totally right will fix

Reply:
fixed https://github.com/mastodon/mastodon/pull/26558/commits/412a2cddcd115734dd89748ea60d1f4207a9cf9a